src/routes/safe/components/Transactions/TxsTable/ExpandedTx/IncomingTxDescription/index.jsx

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react'
+import { withStyles } from '@material-ui/core/styles'
```

Comment:
not required to change, but let's use hooks api for new components (makeStyles)

Reply:
Done. Glad we move onto hooks, thanks!